process/factory/shard/vmContainerFactory.go

```diff
@@ -258,16 +242,6 @@ func (vmf *vmContainerFactory) createInProcessArwenVMByVersion(version config.Ar
 	}
 }
 
-func (vmf *vmContainerFactory) createOutOfProcessArwenVMByVersion(version config.ArwenVersionByEpoch) (vmcommon.VMExecutionHandler, error) {
-	logVMContainerFactory.Debug("createOutOfProcessArwenVM", "version", version)
-	switch version.Version {
-	case "v1.2":
-		return vmf.createOutOfProcessArwenVMV12()
```

Comment:
Delete function create out of process arwen v12

Reply:
done